cclib/bridge/cclib2horton.py

```diff
@@ -0,0 +1,84 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2020, the cclib development team
+#
+# This file is part of cclib (http://cclib.github.io) and is distributed under
+# the terms of the BSD 3-Clause License.
+
+"""Bridge for using cclib data in horton (http://theochem.github.io/horton)."""
+
+import numpy
+from cclib.parser.data import ccData
+from cclib.parser.utils import find_package
+
+# First check horton version
+_old_horton = False
+_found_horton = find_package('horton')
+_found_iodata = find_package('iodata')
+
+if _found_horton: # Detect whether horton 2 is present or not
+    try: # Older horton packages do not have __version__, causing exceptions.
+        from horton import __version__
+    except:
+        _old_horton = True
+    else:
+        if __version__[0] == "2":
+            from horton.io.iodata import IOData
+
+if _found_iodata: # Detect whether iodata (part of horton 3) is present or not; Horton 3 is divided into smaller (sub)packages that each take different functionalities.
```

Comment:
In general, can you put comments like this on the line above of the code they're describing, and also reflow them so they don't go over ~100 characters?

Reply:
> In general, can you put comments like this on the line above of the code they're describing, and also reflow them so they don't go over ~100 characters?

Yes - I've just fixed them and ran `black` on the files that were modified in this PR.